### DISCUSSION TITLE
Is it possible to disable cropper and display only image without stencil

### DISCUSSION THREAD

**Discussion by ramicbenjamin** [advanced-cropper/vue-advanced-cropper#204] [Q&A]
Hello, 

Thank you for creating and maintaining great library! 

I wanted to ask is there ability to disable cropping and just use it as a preview of an image? 

Use case for which I want to use it is that I want to display an image, as a preview before upload, and if user wants to crop it, user needs to click crop button which allows cropping and normal use of the library. I achieved it with having `img` element and displaying an image, and then hiding that element in favor of cropper when user wants to crop, but animations that cropper has are making it glitchy. Video below: 

https://user-images.githubusercontent.com/15839776/179515173-7fd89748-e372-4d76-bc57-c96c17a6fd47.mov

It would be great if I could just pass a prop and say, previewable, or disabled, and it would just give me a preview of the image.

Thanks for your help in advance!

**Comment by Norserium** [comment]
@ramicbenjamin, alas, with the old core your possibilities are pretty limited, but let's try.

Do you want to get the real preview of cropped area or just hide the stencil? As I see, you just hide the stencil on the video. There are plenty ways to do it.

For example, you can play with cropper parameters to get [this](https://codesandbox.io/s/vue-advanced-cropper-preview-h9bclo?file=/src/App.vue).
